### PR TITLE
[as4630-54pe] Fix issue for management port eth2 change to eth0

### PR DESCRIPTION
**Pull request by ec-michael-shih**
#### Why I did it
Our management port is using eth2, but now need to change to using eth0.
#### How I did it
The method I did is using service to trigger shell script to change eth2 link to eth0.
I know the best method is using u-dev rules to name the management interface, but
with the current (accton)code base，I need to add the rules for udev to create the path about /etc/udev/rules.d in "rules"(common code.)
It’s a bit difficult to modify the common code in accton project with little issue. 
#### How to verify it
After the modify, I set eth0 new ipaddress in my local side:
using "ifconfig eth0 192.168.1.12"
and ping 192.168.1.37 is success。

Test log:
root@sonic:# ifconfig eth0 192.168.1.12
root@sonic:#
root@sonic:~# ping 192.168.1.37  (Note: 192.168.1.37my local ipddress)
PING 192.168.1.37 (192.168.1.37) 56(84) bytes of data.
64 bytes from 192.168.1.37: icmp_seq=1 ttl=128 time=2.52 ms
64 bytes from 192.168.1.37: icmp_seq=2 ttl=128 time=1.49 ms
64 bytes from 192.168.1.37: icmp_seq=3 ttl=128 time=1.92 ms
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [x] 201911
- [x] 202006
- [x] 202012

#### Description for the changelog

Add to two file:
sonic-buildimage/platform/broadcom/sonic-platform-modules-accton/as4630-54pe/service/as4630-54pe-platform-handle-mgmt-interface.service
sonic-buildimage/platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+#Due to the hardware design, as4630-54pe use "eth2" instead of "eth0" as management interface.
+#Rename netdev "eth0" and "eth2" to swap original "eth2" to "eth0".
+
+ifconfig eth0 down
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh:6]: can you use udev rule to rename the eth0?

Comment (ec-michael-shih) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh:6]: Hi lguohan:
Yes!! I have tried to use udev rule, and work good.
But, I found out  the all of the project in Accton all use same architecture. 
If I use /etc/udev/rules.d/70-persistent-net.rules to change eth2 to eth0, 
our command code (TOPDIR/platform/broadcom/sonic-platform-modules-accton/debian/rules) need to add a rule to copy udev/rules.d/70-persistent-net.rules.
As our internal-discussion, we do not want to modify common code(ex: rules).
So, can we help me merge this issue into the project. This issue is emergency.

Common code modify Example:
(for mod in $(MODULE_DIRS); do \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} $(KERNEL_SRC)/$(INSTALL_MOD_DIR); \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} usr/local/bin; \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} lib/systemd/system; \
**dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} etc/udev/rules.d;**
cp $(MOD_SRC_DIR)/$${mod}/$(MODULE_DIR)/*.ko debian/$(PACKAGE_PRE_NAME)-$${mod}/$(KERNEL_SRC)/$(INSTALL_MOD_DIR); \
cp $(MOD_SRC_DIR)/$${mod}/$(UTILS_DIR)/* debian/$(PACKAGE_PRE_NAME)-$${mod}/usr/local/bin/; \
cp $(MOD_SRC_DIR)/$${mod}/$(SERVICE_DIR)/*.service debian/$(PACKAGE_PRE_NAME)-$${mod}/lib/systemd/system/; \
**cp $(MOD_SRC_DIR)/$${mod}/udev/* debian/$(PACKAGE_PRE_NAME)-$${mod}/etc/udev/rules.d/;**

Comment (lguohan) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh:6]: can you create an issue in buildimage and assign to accton, to make you folks will come back to use udev rule?

Comment (ec-michael-shih) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh]: > 
> 
> can you create an issue in buildimage and assign to accton, to make you folks will come back to use udev rule?

In our internal discussion ,  supervisor agree the modification in TOPDIR/platform/broadcom/sonic-platform-modules-accton/debian/rules . I will use udev style to solve this issue. Thank you for your suggestion.

**Comment by ec-michael-shih** [comment]
retest this please

**Comment by ec-michael-shih** [comment]
retest this please

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh (hunk 1 above)
Comment (ec-michael-shih) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh]: retest this please

Comment (ec-michael-shih) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh]: retest
 please

Comment (ec-michael-shih) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh]: reset please

**Comment by ec-michael-shih** [comment]
retest this please

**Comment by ec-michael-shih** [comment]
reset please

**Comment by ec-michael-shih** [comment]
reset this please

**Review by ec-michael-shih — COMMENTED** [review]
reset this please

**Comment by ec-michael-shih** [comment]
reset this please

**Comment by ec-michael-shih** [comment]
azpw run

**Pull request closed without merging**